packages/web3-eth-abi/src/utils.ts

```diff
@@ -152,6 +152,10 @@ export const isOddHexstring = (param: unknown): boolean =>
 export const formatOddHexstrings = (param: string): string =>
 	isOddHexstring(param) ? `0x0${param.substring(2)}` : param;
 
+const paramTypeBytes = /^bytes([0-9]*)$/;
```

Comment:
Move them out of the function because they are constants and we do not need to create them every time we execute the function.